src/providers/AuthServerProvider.ts

```diff
@@ -0,0 +1,277 @@
+import { ChainId, ProviderType } from '@dcl/schemas'
+import { ethers } from 'ethers'
+import { Socket, io } from 'socket.io-client'
+import { Authenticator, AuthIdentity, AuthLinkType } from '@dcl/crypto'
+import * as sso from '@dcl/single-sign-on-client'
+import { AuthServerConnector } from '../connectors'
+import { getRpcUrls } from '../configuration'
+
+export class AuthServerProvider {
+  private static authServerUrl: string = ''
+  private static authDappUrl: string = ''
+  private static identityExpirationInMillis: number = 30 * 24 * 60 * 60 * 1000 // 30 days in the future.
+
+  private chainId = ChainId.ETHEREUM_MAINNET
+  private account?: string
+
+  /**
+   * Set the url of the auth server to be consumed by this provider.
+   */
+  static setAuthServerUrl(url: string) {
```

Comment:
These method and variables will change every instance of the `AuthServerProvider` as they're static. Is this the intended behavior?

Reply:
Yes. So they can be configured once for all